web/src/overlays.ts

```diff
@@ -237,8 +261,15 @@ export function close_overlay(name) {
         return;
     }
 
+    if (active_overlay === undefined) {
+        blueslip.error("close_overlay called without checking is_active()");
+        return;
+    }
+
     blueslip.debug("close overlay: " + name);
 
+    const {$element: $active_overlay, close_handler} = active_overlay;
```

Comment:
This feels very hard to read with `active_overlay` and `$active_overlay` variables; I applied this change to address that:
``` diff
@@ -268,20 +267,13 @@ export function close_overlay(name: string): void {
 
     blueslip.debug("close overlay: " + name);
 
-    const {$element: $active_overlay, close_handler} = active_overlay;
-
-    $active_overlay.removeClass("show");
+    active_overlay.$element.removeClass("show");
 
-    $active_overlay.attr("aria-hidden", "true");
+    active_overlay.$element.attr("aria-hidden", "true");
     $(".app").attr("aria-hidden", "false");
     $(".header").attr("aria-hidden", "false");
 
-    if (!close_handler) {
-        blueslip.error("Overlay close handler for " + name + " not properly set up.");
-        return;
-    }
-
-    close_handler();
+    active_overlay.close_handler();
 }
 
 export function close_active(): void {
```

(Also got rid of a check that should be a noop now that we have TypeScript).

Reply:
Yeah looks good to me, Thanks for the change!